Python/pystate.c

```diff
@@ -147,6 +147,24 @@ _PyRuntimeState_ReInitThreads(_PyRuntimeState *runtime)
     PyMemAllocatorEx old_alloc;
     _PyMem_SetDefaultAllocator(PYMEM_DOMAIN_RAW, &old_alloc);
 
+#ifdef HAVE_FORK
```

Comment:
_PyRuntimeState_ReInitThreads() is only called by PyOS_AfterFork_Child(): so only if HAVE_FORK is defined.

I suggest to only defined _PyRuntimeState_ReInitThreads() if HAVE_FORK is defined. Said differently, put the whole function into #ifdef HAVE_FORK ... #endif. Here and where it's prototype in the header file (pycore_runtime.h).

Same remark for _PyImport_ReInitLock() and _PyEval_ReInitThreads().